QuizWhois.Api/Controllers/QuestionController.cs

```diff
@@ -23,22 +24,47 @@ public QuestionController(IQuestionService questionService, IQuestionRatingServi
         }
         
         [HttpPost]
-        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(CreatedResult), StatusCodes.Status201Created)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
-        public async Task<ActionResult> CreateQuestions(List<QuestionModel> questions)
+        public async Task<ActionResult> CreateQuestions(List<QuestionModelRequest> questions)
```

Comment:
lets wrap the list up. request body has to have a root model which includes all lists props etc